### PR TITLE
Google maps api

### PR DESCRIPTION
**Pull request by isamitre**

### CODE REVIEW

File: portfolio/src/main/webapp/html/pages/Map.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <title>Map</title>
+    <link rel="stylesheet" href="../../style.css">
+    <script src="/javascript/script.js"></script>
+  </head>
+  <body>
+    <iframe class="navbar" src="../components/navbar.html"></iframe>
+    <h1 class="center">Map</h1>
+    <br>
+    <!-- <div id="map" </div>
+    <script src="../../javascript/map.js" defer></script> -->
```
Comment (oturner24) [inline portfolio/src/main/webapp/html/pages/Map.html:14]: Can you delete this?

File: portfolio/src/main/webapp/style-map.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* Always set the map height explicitly to define the size of the div
+* element that contains the map. */
+#map {
+  height: 100%;
+}
+/* Optional: Makes the sample page fill the window. */
```
Comment (oturner24) [inline portfolio/src/main/webapp/style-map.css:6]: nit: maybe remove the "Optional:"

File: portfolio/src/main/webapp/html/pages/Map.html (hunk 1 above)
Comment (isamitre) [reply portfolio/src/main/webapp/html/pages/Map.html:14]: yeah

File: portfolio/src/main/webapp/style-map.css (hunk 2 above)
Comment (isamitre) [reply portfolio/src/main/webapp/style-map.css]: done :)

**Pull request merged**